includes/User/MySimsPageBuilder.php

```diff
@@ -44,7 +61,7 @@ public function build_html() {
             '<div class="my-esim-page-wrapper">
                     <div class="left-menu">
                         <div>
-                            <p class="left-menu-title">Available eSIMs</p>
+                            <p class="left-menu-title">'.$this->t('my.esims.available.esims').'</p>
```

Comment:
please add some spacing between the string concatenation, this will trigger the WP checks unfortonately

Reply:
updated, @MiroIsikiyski